AutowareAuto/src/control/trajectory_follower/src/mpc_utils.cpp

```diff
@@ -329,17 +329,21 @@ int64_t calcNearestIndex(
   if (traj.points.empty()) {
     return -1;
   }
-  const float64_t my_yaw = tf2::getYaw(self_pose.orientation);
+  const float64_t my_yaw = 0.0; //tf2::getYaw(self_pose.orientation);
+  std::cerr<< "my_yaw:  "<<my_yaw << std::endl;
   int64_t nearest_idx = -1;
   float64_t min_dist_squared = std::numeric_limits<float64_t>::max();
   for (size_t i = 0; i < traj.points.size(); ++i) {
     const float64_t dx = self_pose.position.x - static_cast<float64_t>(traj.points.at(i).x);
     const float64_t dy = self_pose.position.y - static_cast<float64_t>(traj.points.at(i).y);
     const float64_t dist_squared = dx * dx + dy * dy;
+    std::cerr<< "dist_squared:  "<<dist_squared<<std::endl;
```

Comment:
after the integration is done, it is probably better to remove these cerr prints, because I imagine they will be printing at 30hz which is too much.